python/ray/services.py

```diff
@@ -165,6 +165,31 @@ def all_processes_alive(exclude=[]):
     return True
 
 
+def hostname_to_ip_address(address):
```

Comment:
I'd also update the function name to be more general, since the code has evolved to support both FQDNs and IPs. I think `address_to_ip()` is a good candidate. Another option is `resolve_address()` (since the output isn't necessarily just an IP, but could be an IP:PORT string).

Reply:
Fixed.